app.js

```diff
@@ -0,0 +1,120 @@
+let books = [
```

Comment:
I would like you to fetch this list from our API – https://bookshop-api.mirkwood.dev/books/